crates/solidity/outputs/npm/crate/Cargo.toml

```diff
@@ -1,18 +1,29 @@
 [package]
-name = "solidity_npm_crate"
+name = "slang_solidity_node_addon"
```

Comment:
I don't feel that strong about the rename, but it never ceased to confuse me why we make that distinction and that `solidity_npm_crate` is effectively `slang_solidity` compiled as cdylib, so this makes it easier on my brain :sweat_smile: . See also https://github.com/NomicFoundation/slang/pull/745/commits/8853229c8f8db1966d4a4926b5ea3fd79afc21f7

Reply:
Please see my comment on the root `Cargo.toml`.